js/packages/common/src/contexts/meta/loadAccounts.ts

```diff
@@ -972,7 +979,7 @@ const pullMetadataByCreators = (
     }
   }
 
-  return Promise.all(additionalPromises);
+  return Promise.all(additionalPromises).then(done);
```

Comment:
It's just creating a new promise from the one that is already there that will call `done` before resolving, so yes I think it is safe. `Promise.all` resolves an array of resolved promises, however those results are `void` and the caller is not using the results. Otherwise we'd have gotten a TypeScript type error after this change.